packages/node/src/retryHandler.ts

```diff
@@ -103,10 +103,22 @@ export class RetryHandler implements RetryClass {
     return prunedEvents;
   }
 
-  private _getPayload(events: readonly Event[]): Payload {
+  private _getPayloadOptions(): Partial<Payload> {
```

Comment:
curious why this `_getPayloadOptions` but it returns a `Payload`? I think it might make sense to create the `PayloadOption` interface and pass that or change the naming of this function

Reply:
good call @kelvin-lu, oversight on my part.  Ended up introducing a `PayloadOptions` interface.